x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -188,13 +223,28 @@ function NodeDetailLink({ name, item }: { name: string; item: ProcessTableView }
           )}
         </EuiBadge>
       ) : (
-        <>
+        <StyledButtonTextContainer>
           <CubeForProcess
             running={!isTerminated}
+            isOrigin={isOrigin}
             data-test-subj="resolver:node-list:node-link:icon"
           />
-          <span data-test-subj="resolver:node-list:node-link:title">{name}</span>
-        </>
+          <StyledLabelContainer>
+            {isOrigin && (
+              <StyledAnalyzedEvent
+                color={descriptionText}
+                data-test-subj="resolver:node-list:node-link:origin"
+              >
+                {i18n.translate('xpack.securitySolution.resolver.panel.table.row.originLabel', {
```

Comment:
This one also seems unused.

Reply:
This is for translations? Updated the name though to be analyzedEvent